src/watchdog_fsevents.c

```diff
@@ -717,14 +741,21 @@ watchdog_read_events(PyObject *self, PyObject *args)
 #endif
 
     /* Allocate information and store thread state. */
-    value = PyDict_GetItem(thread_to_run_loop, emitter_thread);
+    switch (PyDict_GetItemRef(thread_to_run_loop, emitter_thread, &value)){
+        case -1:
+            return NULL;
+        case 0:
+            run_loop_ref = CFRunLoopGetCurrent();
+            value = PyCapsule_New(run_loop_ref, NULL, watchdog_pycapsule_destructor);
+            PyDict_SetItem(thread_to_run_loop, emitter_thread, value);
+            Py_INCREF(emitter_thread);
```

Comment:
I guess it's always worked this way technically but I don't understand why there's an incref here but not the other cases.